src/multi-select/tp-multi-select.ts

```diff
@@ -106,9 +106,14 @@ export class TPMultiSelectElement extends HTMLElement {
 	get value(): string[] {
 		const value: string[] = [];
 
-		const selectedOptions: NodeListOf<HTMLOptionElement> | null = this.querySelectorAll( 'select option[selected]' );
-		selectedOptions?.forEach( ( option: HTMLOptionElement ) => value.push( option.value ) );
-
+		const selectedOptions: NodeListOf<HTMLOptionElement> | null = this.querySelectorAll( 'tp-multi-select-option' );
```

Comment:
@i-am-chitti Since the ultimate value of the field comes from the hidden `select` field - it would be best if that is the source of truth for values instead of `tp-multi-select-option`. This is how it was before this update

Reply:
Have kept it the same.